### PR TITLE
revert change to env var

### PR DESCRIPTION
**Pull request by privateducky**

### CODE REVIEW

File: shell/probe/raindrop.ps1 [hunk 1]
```diff
@@ -29,7 +30,6 @@ $Dat = ""
 
 while ($true) {
     try {
-        $VstDir = New-Item -ItemType Directory -Path ".vst" -Force
```
Comment (khyberspache) [inline shell/probe/raindrop.ps1:32]: Keep this, just pass the full path here, leave everything else the same

**Review by khyberspache — CHANGES_REQUESTED** [review]
One adjustment

File: shell/probe/raindrop.ps1 (hunk 1 above)
Comment (privateducky) [reply shell/probe/raindrop.ps1:32]: ha, got this just as i did.. can you check if my change should work?

Comment (khyberspache) [reply shell/probe/raindrop.ps1:32]: Like use $Dir as the argument to this line, keep everything else the same

Comment (privateducky) [reply shell/probe/raindrop.ps1:32]: did i do it...?

Comment (khyberspache) [reply shell/probe/raindrop.ps1:32]: No lol you need to capture this to a variable, just revert all your changes and add the one line at the top to pull the directory from the environment

Comment (privateducky) [reply shell/probe/raindrop.ps1:32]: oh - i THINK i see... like this?

Comment (privateducky) [reply shell/probe/raindrop.ps1:32]: i totally got it, didn't i?

Comment (khyberspache) [reply shell/probe/raindrop.ps1:32]: You did :)

**Pull request merged**